src/main/java/net/sf/rails/ui/swing/StatusWindow.java

```diff
@@ -137,7 +144,21 @@ public void initMenu() {
         moderatorMenu.setMnemonic(KeyEvent.VK_M);
         specialMenu.setMnemonic(KeyEvent.VK_S);
 
-        ActionMenuItem actionMenuItem = new ActionMenuItem(LocalText.getText("SAVE"));
+        ActionMenuItem actionMenuItem = new ActionMenuItem(NEW_CMD);
+        actionMenuItem.setActionCommand(NEW_CMD);
+        actionMenuItem.addActionListener(this);
+        actionMenuItem.setEnabled(false);
+        actionMenuItem.setPossibleAction(new GameAction(gameUIManager.getRoot(), GameAction.Mode.NEW));
+        fileMenu.add(actionMenuItem);
+
+        actionMenuItem = new ActionMenuItem(LOAD_CMD);
+        actionMenuItem.setActionCommand(LOAD_CMD);
+        actionMenuItem.addActionListener(this);
+        actionMenuItem.setEnabled(true);
+        actionMenuItem.setPossibleAction(new GameAction(gameUIManager.getRoot(), GameAction.Mode.LOAD));
+        fileMenu.add(actionMenuItem);
+
+        actionMenuItem = new ActionMenuItem(LocalText.getText("SAVE"));
```

Comment:
I would like to have a shortkey for those actions as well :)

Reply:
Noted-- have a menu change PR coming, will try to find appropriate keys